packages/cosmos/library/types/google/protobuf/descriptor.ts

```diff
@@ -558,20 +571,26 @@ export interface MessageOptions {
 	 * @deprecated
 	 */
 	deprecated_legacy_json_field_conflicts?: boolean;
-	/** Any features defined in the specific edition. */
+	/**
+	 * Any features defined in the specific edition.
+	 * WARNING: This field should only be used by protobuf plugins or special
+	 * cases like the proto compiler. Other uses are discouraged and
+	 * developers should rely on the protoreflect APIs for their client language.
+	 */
 	features?: FeatureSet;
 	/** The parser stores options it doesn't recognize here. See above. */
 	uninterpreted_option: UninterpretedOption[];
 }
 
 export interface FieldOptions {
 	/**
+	 * NOTE: ctype is deprecated. Use `features.(pb.cpp).string_type` instead.
 	 * The ctype option instructs the C++ code generator to use a different
 	 * representation of the field than it normally would.  See the specific
 	 * options below.  This option is only implemented to support use of
 	 * [ctype=CORD] and [ctype=STRING] (the default) on non-repeated fields of
-	 * type "bytes" in the open source release -- sorry, we'll try to include
-	 * other types in a future version!
+	 * type "bytes" in the open source release.
+	 * TODO: make ctype actually deprecated.
```

Comment:
Its a bit confusing to understand the information here comparing L587 to this line